bindata/network/openshift-sdn/controller.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   selector:
     matchLabels:
       app: sdn-controller
-  strategy:
+  updateStrategy:
```

Comment:
Thanks to there being no UpdateStrategy, it uses the default, which is to surge to N+1. That's fine in this case, since sdn-controller has locking.

Probably better to just drop the field and keep it as-is.